hera_mc/cm_sysutils.py

```diff
@@ -776,6 +780,8 @@ def node_info(node_num="active", session=None):
             except KeyError:
                 info[snp] = []
         notes = hu.get_notes(wr, state="all", return_dict=True)
+        if not isinstance(info[node], dict):
+            continue
```

Comment:
@david-deboer can you add a testing for this line please?

Reply:
@david-deboer ping on this

Reply:
That line is clearly not needed, since info[node] is defined right there and has to be a dict.